### DISCUSSION TITLE
Hosting Unleash on 2 Ports in docker

### DISCUSSION THREAD

**Discussion by deepak-grover9** [Unleash/unleash#5682] [General]
Is there a way to host unleash on both HTTP and HTTPS port with different hosts. My application is built using docker and nginx.

**Comment by daveleek** [comment]
Hello @deepak-grover9, apologies for us not having gotten back to you sooner.
Depending on your setup and how your containers are built I would look into [Nginx configuration for SSL](https://nginx.org/en/docs/http/configuring_https_servers.html#single_http_https_server) and potentially also exposing multiple ports from the docker container (In your docker file you can use the expose verb and add multiple ports separated by space character). Hope this helps

**Reply by deepak-grover9** [reply]
Hi This issue has been resolved by exposing the port in unleash.start(). It can be closed.

**Reply by daveleek** [reply]
Great, happy to hear that! Thank you @deepak-grover9, I'll close it!
